.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Fixes #
+
+#### Changes proposed in this Pull Request:
```

Comment:
I think just `#### Proposed changes` is enough for here. You can drop the trailing `:` since this is a heading.